extensions/amp-render/1.0/amp-render.js

```diff
@@ -268,24 +267,33 @@ export class AmpRender extends BaseElement {
                 return Services.bindForDocOrNull(this.element);
               })
               .then((bind) => {
-                const bindingAttrValue = this.element.getAttribute('binding');
+                if (!bind) {
+                  throw new Error();
+                }
+                const bindingValue = this.element.getAttribute('binding');
                 return bind.rescan([element], [], {
                   'fast': true,
                   'update':
                     // update should be true if:
                     // 1. no binding is specified (default is binding=always) or
```

Comment:
Are we supporting `always`?

Reply:
Yes for now, to keep parity with amp-list. I have cookbook queries to see the usage & perhaps we can make an informed decision once we have some data.

Reply:
Looked at the [cookbook query](https://cookbook.corp.google.com/recipe/549562) for last couple of days, we have ~1M URLs that explicitly specify `binding="always"` for amp-list, so I'm in favor of supporting the same for amp-render.

Reply:
Based on offline discussion, dropping support for `always`. `refresh` will be the default value.

Reply:
As per offline discussion `always` is supported, but with a warning.